build_docs.sh

```diff
@@ -16,9 +16,9 @@ cd ../..
 cd Docs/sphinx_documentation
 
 echo "Build the Sphinx documentation for Amrex."
-make PYTHON="python3" latexpdf
+make SPHINXOPTS='-v -W --keep-going' PYTHON="python3" latexpdf
 mv build/latex/amrex.pdf source/
-make PYTHON="python3" html &> make_source_html.out
+make SPHINXOPTS='-v -W --keep-going' PYTHON="python3" html &> make_source_html.out
```

Comment:
I think we should remove `&> make_source_html.out` so that we can actually see the output and error.

Reply:
We also need to build `html` first to treat warnings as errors, if we ignore the warnings from `latexpdf`. Or we do `make clean` before `make html`.

Reply:
The current code breaks. Now let me fix the table again.